app/common/directives/first-time-config.html

```diff
@@ -48,13 +48,13 @@ <h1 translate>settings.customize_your_new_deployment</h1>
         <h3 class="edit" translate>settings.add_first_post</h3>
         <div class="divider padded"></div>
         <p translate>settings.add_first_post_subtitle</p>
-        <a class="button button-create" ui-sref="postCreate" ng-click="closeModal()">
+        <a class="button button-create" role="button" ui-sref="postCreate" ng-click="closeModal()">
             <span translate>settings.add_first_post</span>
         </a>
         <div class="divider padded"></div>
         <p translate>settings.invite_people</p>
-        <a class="button button-primary" ui-sref="settings.users" ng-click="closeModal()">
+        <a class="button button-primary" role="button" ui-sref="settings.users" ng-click="closeModal()">
             <span translate>settings.add_new_users</span>
         </a>
     </div>
-</modal>
+</modal>
```

Comment:
Hey! Can you remove this change. Just add a newline after `</modal>` and it should be fine.